src/main/jte/bookings/create/conflict.jte

```diff
@@ -30,7 +30,7 @@
                             <p class="list-col-wrap">
                                 ${booking.getDescription().isBlank() ? context.lookup("bookings.description.none") : booking.getDescription()}
                             </p>
-                            <a class="btn btn-square btn-ghost" href="/${booking.getRoom().getId()}/bookings/${booking.getId()}" target="_blank" aria-label="${context.localize("bookings.view")}">
+                            <a class="btn btn-square btn-ghost" href="/${booking.getRoom().getId()}/bookings/${booking.getId()}" target="_blank" aria-label="${context.localize("bookings.view")}" title="${context.localize("bookings.view")}">
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.